src/Features/CSharp/Portable/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs

```diff
@@ -40,10 +40,7 @@ internal sealed class CSharpAddBracesDiagnosticAnalyzer : DiagnosticAnalyzer, IB
                 SyntaxKind.LockStatement);
 
         public override void Initialize(AnalysisContext context)
```

Comment:
Should this be made a `CodeStyleAnalyzer` and override InitializeWorker instead?